app/portainer/services/api/endpoint.service.ts

```diff
@@ -0,0 +1,50 @@
+import {
```

Comment:
shouldn't this file be moved to `app/portainer/environments` ? or is it here for backwards compatibility for now?

Reply:
backward compatibility